Makefile

```diff
@@ -86,8 +86,11 @@ cluster-up:
 cluster-down:
 	./cluster/down.sh
 
-cluster-sync:
```

Comment:
Can we please leave the `cluster-sync` command ? and let him use the `cluster-install-operator` ?

Reply:
why? i will add cluster-install-registry, that will not be compatible with cluster-install-operator, so i'd rather keep them symmetrical. any specific reason to keep it? if there are multiple ways to install the operator, who says which one should have the privilege to be under cluster-sync?

Reply:
I'm fine with cluster-sync-*

Reply:
I think we should have the `cluster-install-operator` running when we call the `cluster-sync` to be able to continue fast development.

And for the why to leave the `cluster-sync` I think most of our project have the same convention of `make cluster-up && make cluster-sync && make functest && make cluster-down`
So I will like to save this.

It's easy when you work on multiple projects under the kubevirt org to use the same commands to create and run the code.